conf/msg_conf/map_msg.conf

```diff
@@ -720,6 +720,15 @@
 695: Rebellion
 696: Oktoberfest
 697: Summoner
+698: Baby Summoner
+699: Baby Ninja
+700: Baby Kagerou
+701: Baby Oboro
+702: Baby Taekwon
+703: Baby Star Gladiator
+704: Baby Soul Linker
+705: Baby Gunslinger
+706: Baby Rebellion
```

Comment:
Values 700+ won't work. You're now overwriting the VIP atcommand messages. The 753-899 range is free though. Don't forget to update all the other language messages as well to reflect the changes.